cmd/spire-server/cli/entry/util.go

```diff
@@ -91,6 +95,34 @@ func printEntry(e *common.RegistrationEntry) {
 	fmt.Println()
 }
 
+// ParseFile parses JSON represented RegistrationEntries
+// if path is "" or "*" read JSON from STDIN
+func ParseFile(in io.Reader, path string) ([]*common.RegistrationEntry, error) {
```

Comment:
Personally I think this can be made to:
```
func ParseFile(path string) ([]*common.RegistrationEntry, error) {
    return parseFile(os.Stdin, path)
}

func parseFile(stdin io.Reader, path string) ([]*common.RegistrationEntry, error) {
    ...
}
```

This provides a nice public API (the consumer doesn't need to pass `os.Stdin`, which is somewhat of an implementation detail), which also giving you the testability I believe you're looking for.

Reply:
Thanks! Looks good! 

6096a68